backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/steps/DataTransformation.kt

```diff
@@ -1,19 +1,35 @@
 package app.ehrenamtskarte.backend.stores.importer.steps
 
 import app.ehrenamtskarte.backend.stores.importer.PipelineStep
-import app.ehrenamtskarte.backend.stores.importer.matchesNa
 import app.ehrenamtskarte.backend.stores.importer.types.ImportAcceptingStore
-import app.ehrenamtskarte.backend.stores.importer.types.LbeAcceptingStore
 import org.slf4j.Logger
 
 class DataTransformation(private val logger: Logger) : PipelineStep<List<ImportAcceptingStore>, List<ImportAcceptingStore>>() {
 
-    override fun execute(input: List<ImportAcceptingStore>) = input.map {
-        val pc = it.postalCode
-        it.copy(
-            categoryId = if (it.categoryId == 99) 9 else it.categoryId,
-            postalCode = if (pc != null && matchesNa(pc)) "" else pc
+
+
+    override fun execute(input: List<ImportAcceptingStore>) = input.map { store ->
+        val pc = if (store.postalCode != null) findPostalCode(store.postalCode) else ""
+        store.copy(
+            categoryId = if (store.categoryId == 99) 9 else store.categoryId,
+            postalCode = pc
         )
     }
 
+    companion object {
+
+        private const val POSTAL_CODE_LENGTH = 5
```

Comment:
We could still support 4-long postal codes :thinking: 

Rolf is greeting! (see the 5 fingers :P)

![image](https://user-images.githubusercontent.com/905221/121388938-2f1c1880-c94c-11eb-8bf2-16aea638bd76.png)